IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MaintenanceInformation/DocPropertySet.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MaintenanceInformation" UniqueId="a5ccf5dc-9c2d-8761-9e7d-06206c294ae5" ApplicableType="IfcDistributionElement,IfcRail/RAIL,IfcRail/RACKRAIL,IfcCovering/INSULATION,IfcSlab/BASESLAB,IfcCourse,IfcElementAssembly/TRACKPANEL,IfcElementAssembly/DILATATIONPANEL,IfcDiscreteAccessory/RAIL_LUBRICATION,IfcMechanicalFastener/RAILFASTENING,IfcDiscreteAccessory/RAILPAD,IfcMember/TIEBAR,IfcDiscreteAccessory/SLIDINGCHAIR,IfcTrackElement/SLEEPER,IfcMechanicalFastener/ANCHORBOLT" PropertySetType="PSET_OCCURRENCEDRIVEN">
+	<Properties>
+		<DocProperty xsi:nil="true" href="InstallationDate_2k4h6AVrL8EBlzWRArJDjn" />
+		<DocProperty xsi:nil="true" href="ManufacturingDate_1Jp6hK45Bou8c2TZpfomg" />
+		<DocProperty xsi:nil="true" href="DisassemblyDate_21F9eF2br0OPwbIT8YD7C" />
+		<DocProperty xsi:nil="true" href="PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv" />
+		<DocProperty xsi:nil="true" href="MeanTimeBetweenFailure_2asqCvmc15PhVbb834HHPQ" />
+		<DocProperty xsi:nil="true" href="AssessmentFrequency_3Y5J0kLVfDHfMH8QTwre7" />
+		<DocProperty xsi:nil="true" href="LastMaintenanceDate_0QPatCUfXDS82qcvRNRkbK" />
+		<DocProperty xsi:nil="true" href="NextMaintenanceDate_1eqCcwujP6M9SVEXqgrvLQ" />
+		<DocProperty xsi:nil="true" href="RepairingDate_1fKpvOEozEKQHboTTf4_f7" />
+		<DocProperty xsi:nil="true" href="IsMaintainedBy_3uo_1qw9T0jubm1OgmJ4Gx" />
+		<DocProperty xsi:nil="true" href="MaintenanceType_1c7wEh3lzAPgRLC8A468u8" />
+		<DocProperty xsi:nil="true" href="MeanTimeBetweenFailures_0ULD8MZJr07eC4LRLixScX" />
+		<DocProperty xsi:nil="true" href="MeanTimeToRepair_3wO8ns0az0euk3DIe06ndR" />
+		<DocProperty xsi:nil="true" href="IsRepairedBy_0FyZMrun94iupog1HstMc" />
+		<DocProperty xsi:nil="true" href="RepairingContent_2uB5y1E3H1zuDscjqI6Rjw" />
+		<DocProperty xsi:nil="true" href="ServiceCycle_01ShZOXG5BLO_Gv105Bz_e" />
+		<DocProperty xsi:nil="true" href="ServiceLevel_1dJezT8M5D1R8Dz2lHSt2y" />
+		<DocProperty xsi:nil="true" href="StorageCondition_2fY6Ixwun9OeL0AdKAOGuo" />
+	</Properties>
```

Comment:
We need to review this along side the other installation, maintance & Asset PSets from other pull requests, i cannot remember the conclusion we came to.